packages/toolkit/.changes/next-release/Feature-76bb396d-6268-4d24-9373-4c7c0e91cbc4.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Step Functions: Upgrade amazon-states-language-service to 1.13. This new version adds support for JSONata and Variables."
```

Comment:
is "JSONata" a typo?

Reply:
Not a typo, it's a new json language feature that we've incorporated into StepFunctions service
https://aws.amazon.com/blogs/compute/simplifying-developer-experience-with-variables-and-jsonata-in-aws-step-functions/